src/Components/Navbar.js

```diff
@@ -13,28 +13,29 @@ export default class Navbar extends Component {
   render() {
     return (
       <nav className="Navbar">
-        <div class="filter-buttons">
-          <h2 class="title">GAMEFLIX</h2>
-          <div class="filters-column-one">
-            <p class="reset-games" data-nav="resetall" onClick={this.props.resetAllGames}>All Games</p>
-            <p class="filter-boardgames" data-nav="resetboard" onClick={this.props.resetAllGames}>Board Games</p>
+        <div className="main-nav">
```

Comment:
Needed to add this div wrapping the default navbar so that the advanced filter bar didn't popup incorrectly because of the media queries.